cli/js/signal_test.ts

```diff
@@ -126,25 +126,30 @@ if (Deno.build.os === "win") {
 
     assertEquals(c, 3);
 
-    clearTimeout(t);
+    clearInterval(t);
+    // Defer for a moment to allow interval promise to resolve
+    await defer(20);
```

Comment:
This is bad - because our internal timer implementation uses promises; after clearing an interval, one actually needs to wait until timer promise resolves. Otherwise that's considered a leak because timers are based on "ops".

Reply:
Can you add a TODO here explaining the situation? We should adjust the interval implementation to avoid this.

Reply:
Added more description and FIXME in `cli/js/timers.ts`